modules/linear_boltzmann_solvers/executors/lbs_steady_state.cc

```diff
@@ -18,14 +18,13 @@ SteadyStateSolver::GetInputParameters()
 {
   InputParameters params = opensn::Solver::GetInputParameters();
 
-  params.SetGeneralDescription(
-    "Generalized implementation of a steady state solver. This solver calls"
-    " the Across-Groupset (AGS) solver for the lbs-data block.");
+  /* clang-format offf */
```

Comment:
```suggestion
  /* clang-format off */
```
On this note, is it clang-format that is adding blank lines in your PRs? It doesn't do that on my end.

Reply:
No, I did that. But, clang-format doesn't consistently align arguments which makes things difficult to read, IMHO. I manually formatted and guarded it.

Reply:
Nice catch, btw.